helm/chaos-mesh/templates/controller-manager-rbac.yaml

```diff
@@ -24,6 +24,9 @@ metadata:
     app.kubernetes.io/component: controller-manager
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
 rules:
+  - apiGroups: [ "" ]
+    resources: [ "endpoints" ]
+    verbs: [ "get", "list", "watch" ]
```

Comment:
This could be removed from the `ClusterRole` and only requires a `Role`. The main problem is that:

1. I want to use the cached client of `controller-runtime` for endpoint.

2. The cache informer scope can only be set globally for the manager (as shown in this [PR](https://github.com/kubernetes-sigs/controller-runtime/pull/136)). 

3. We cannot set manager cache namespace globally into `chaos-testing` (or another namespace), because we want to get pods through cached client.